lib/bundlex/toolchain/common/unix/os_deps.ex

```diff
@@ -173,32 +157,19 @@ defmodule Bundlex.OSDeps do
       String.split(url, "/")
       |> Enum.at(-1)
       |> String.split(".")
-      |> Enum.reject(&(&1 in ["tar", "xz"]))
+      |> Enum.reject(&(&1 in ["tar", "xz", "gz"]))
```

Comment:
I've bumped into https://github.com/ricn/zarex, maybe it's worth considering?

Reply:
good idea, done